README.md

```diff
@@ -2,14 +2,28 @@
 
 Assists you to programmatically create, edit and delete Links on Instamojo in PHP.
 
-**Note**: If you're using this wrapper with our sandbox environment `https://test.instamojo.com/` then you should pass `'https://test.instamojo.com/api/1.1/'` as third argument to the `Instamojo` class while initializing it. API key and Auth token for the same can be obtained from https://test.instamojo.com/developers/ (Details: [Test Or Sandbox Account](https://instamojo.zendesk.com/hc/en-us/articles/208485675-Test-or-Sandbox-Account)).
+**Note**: 
+* If you are using other version of instamojo-php you can refer to their docs from appropriate link below
+  * [v1.1](docs/v1.1)
+
+* If you're using this wrapper with our sandbox environment `https://test.instamojo.com/` then you should pass `true` as third argument to the `Instamojo` class while initializing it. `client_id` and `client_secret` token for the same can be obtained from https://test.instamojo.com/developers/ (Details: [Test Or Sandbox Account](https://instamojo.zendesk.com/hc/en-us/articles/208485675-Test-or-Sandbox-Account)).
```

Comment:
Rephrase the sentence so that it is easier to understand..
* If you're using this wrapper with our sandbox environment `https://test.instamojo.com/` then pass third argument of `Instamojo::init' method as 'true'. `client_id` and `client_secret` token for the same can be obtained from https://test.instamojo.com/developers/ (Details: [Test Or Sandbox Account](https://instamojo.zendesk.com/hc/en-us/articles/208485675-Test-or-Sandbox-Account)).